### DISCUSSION TITLE
PointRend training on custom dataset problem

### DISCUSSION THREAD

**Discussion by rodneytai** [facebookresearch/detectron2#3264] [Q&A]
Recently I want to train my dataset on PointRend in Colab/Local, however, Colab and Local both return Assertion Error and Unknow Pooler Type.  The following is my config and errors.
This error happened in Colab:
![image](https://user-images.githubusercontent.com/50794396/126285271-eb81865a-1837-452f-b15d-96ba46a0693b.png)
![image](https://user-images.githubusercontent.com/50794396/126285458-bdf22ea0-4963-4720-bfba-f608aab06a16.png)

This error happened in local:
![image](https://user-images.githubusercontent.com/50794396/126285524-d46a951d-a987-4a67-8cd5-113ed91b27f5.png)

Any ideas how to fix this ? Thanks in advanced.

**Comment by ppwwyyxx** [comment]
It appears that you install an old version of detectron2 but puts a newer version in `detectron2_repo/`. They are supposed to have the same version.
